src/hooks/useMessagesLane.ts

```diff
@@ -18,7 +18,7 @@ import { ApiPromise } from '@polkadot/api';
 import BN from 'bn.js';
 import { useEffect, useState } from 'react';
 
-import useLaneId from '../hooks/useLaneId';
+import { useLaneId } from '../hooks';
```

Comment:
Again, I'm not sure what the gain actually is. Like I can accept if we decide to never use default exports (I'm a big fan of that approach actually), but having "hub" files is not really something that I fully understand. Is it about saving few keystrokes to avoid repeating `useLaneId` on both sides?

Reply:
It mainly comes down to this:
From:
```
import Accounts from '../components/Accounts';
import CustomCall from '../components/CustomCall';
import DashboardCard from '../components/DashboardCard';
import Remark from '../components/Remark';
import SnackBar from '../components/SnackBar';
import Transactions from '../components/Transactions';
import Transfer from '../components/Transfer';
```
To:
```
import { Accounts, CustomCall, DashboardCard, Remark, SnackBar, Transactions, Transfer } from '../components';
```

It is not a matter of right or wrong - but mostly of a decision making in the team on what approach is preferable
At the end of the day you are right and I am right and Hector is right :)